src/components/Swap/validation.ts

```diff
@@ -19,7 +19,7 @@ export const validateSlippage = (value: string): string | undefined => {
   const schema = fixedNumber(2).min(0).max(100);
   try {
     schema.validateSync(Number(value || 0));
-  } catch (e) {
+  } catch (e: any) {
```

Comment:
Please add `noImplicitAny` rule to the tsconfig and don't use `any` type in the project

Reply:
If I didn't implicitly define the `e` variable as any, the following error occurs.
`Object is of type 'unknown'.  TS2571`

Reply:
Find a solution without `any` please)